app/routes.js

```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
   let assessmentMarkingRegex = new RegExp(/Marking assessments/i)
   let assessmentModeratingRegex = new RegExp(/Moderating assessments/i)
   let assessmentDesigningRegex = new RegExp(/Designing and setting assessments/i)
+  let assessmentSeniorRegex = new RegExp(/Designing and setting assessments/i)
 
   let isMarking = assessmentMarkingRegex.test(assessmentExpertise)
   let isModerating = assessmentModeratingRegex.test(assessmentExpertise)
   let isDesigning = assessmentDesigningRegex.test(assessmentExpertise)
 
+  if (isMarking == true) {
+    // this is to use in the nunjucks view
+    req.session.data.isMarking = true 
+  } else {
+    isMarking = false
+  }
+
+  if (isModerating == true) {
+    // this is to use in the nunjucks view
+    req.session.data.isModerating = true 
+  } else {
+    isModerating = false
+  }
+
+  if (isDesigning == true) {
+    // this is to use in the nunjucks view
+    req.session.data.isDesigning = true 
+  } else {
+    isDesigning = false
+  }
+
```

Comment:
Need to add the one for Reviewing 

`if (isReviewing == true) {
    // this is to use in the nunjucks view
    req.session.data.isReviewing = true 
  } else {
    isReviewing = false
  }`